packages/aws-library/src/aws_library/ec2/_errors.py

```diff
@@ -0,0 +1,33 @@
+from pydantic.errors import PydanticErrorMixin
+
+
+class EC2RuntimeError(PydanticErrorMixin, RuntimeError):
```

Comment:
Why not using our `from models_library.errors_classes import OsparcErrorMixin`

Reply:
true that

Reply:
will do another PR to change that in all AWS clients